api/src/formatters/android-xml-spec.ts

```diff
@@ -0,0 +1,14 @@
+import { loadFixture, simpleFormatFixture } from './fixtures';
+import { androidXmlExporter, androidXmlParser } from './android-xml';
+
+test('should parse gettext files', async () => {
```

Comment:
This test description seems to be wrong.

Reply:
Fixed, thanks!